runtime/expr/function/ztests/regexp-replace.yaml

```
@@ -1,8 +1,9 @@
 zed: "yield regexp_replace(in, re, new)"
 
 input: |
-  {in:"-ab-axxb-",re:"a(x*)b",new:"T"}
+  {in:"-ab-axxb-",re:"ax*b",new:"T"}
```

Keep this as is so it remains the same as the expression on the next line.
```suggestion
  {in:"-ab-axxb-",re:"a(x*)b",new:"T"}
```